app/views/static_pages/contact.html.erb

```diff
@@ -0,0 +1,9 @@
+<% provide(:title, 'Contact')%>
+<!DOCTYPE html>
+<html>
```

Comment:
Please review our code in TWITTER clone or Parent-Child relationship lessons regarding Ruby on Rails .html.erb files